am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GrantAccessIntegrationTest.java

```diff
@@ -39,9 +39,9 @@ void whenCreatingResourceAccessResourceAccessAppearsInDatabase() {
 
     @Test
     void whenCreatingResourceAccessMultipleEntriesAppearInDatabase() {
-        Map<JsonPointer, Set<Permission>> multipleAttributePermissions = new ConcurrentHashMap<>();
-        multipleAttributePermissions.put(JsonPointer.valueOf(""), EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS);
-        multipleAttributePermissions.put(JsonPointer.valueOf("/name"), EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS);
+        Map<JsonPointer, Set<Permission>> multipleAttributePermissions =
```

Comment:
Can we reformat it slightly to look more or less like:

```
Map<JsonPointer, Set<Permission>> multipleAttributePermissions = ImmutableMap.of(
    JsonPointer.valueOf(""), EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS,
	JsonPointer.valueOf("/name"), EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS
);
```

It is slightly easier to read IMO since values are aligned same way.

Reply:
👍